src/components/infoTooltips/PriceImpactTooltip.tsx

```diff
@@ -0,0 +1,49 @@
+import { Trans } from '@lingui/macro';
+import { Box, Skeleton } from '@mui/material';
+
+import { FormattedNumber } from '../primitives/FormattedNumber';
+import { TextWithTooltip } from '../TextWithTooltip';
+
+interface PriceImpactTooltipProps {
+  loading: boolean;
+  priceImpact: string;
+}
```

Comment:
Should this also extend `TextWithTooltipProps` like `SlippageTooltip` does?

Reply:
yeah that's probably a good idea